setup.py

```diff
@@ -107,7 +107,7 @@ def copy_file (self, filename, dirname):
     ],
     cmdclass=cmdclass,
     command_options=command_options,
-    requires=['pexpect', 'python_magic', 'pyyaml']
+    install_requires=['pexpect', 'python_magic', 'pyyaml']
```

Comment:
AFAIK, this would be correct for `setuptools`, but sos to date still uses `distutils` where `requires` is the keyword to use. I believe `distutils` will be removed in `python-3.12`, but at the moment we still use it.

In order for this change to be complete, we'd need to update the entire `setup.py` to use `setuptools`. Which is a PR we'd be happy to accept, of course.